microsoftgraph/microsoft-graph-explorer
# Feat/hide share btn

## Overview

Graph Explorer does not support sharing destructive api calls. This PR hides the share button when a user selects a destructive HTTP verb.

### Demo

## GET

![with_get](https://user-images.githubusercontent.com/12011447/47076287-10027280-d207-11e8-8ef0-86be0d7317b5.JPG)

## POST

![with_post](https://user-images.githubusercontent.com/12011447/47076313-198bda80-d207-11e8-8059-96fd26d08b00.JPG)


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Navigate to Graph Explorer
* Log In
* Change the HTTP verbs
* Observe the share buttons
